developer-docs-site/docs/concepts/node-networks-sync.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: "Node Networks and Sync"
-slug: "basics-node-networks-sync"
+slug: "node-networks-sync"
 ---
 
 # Node Networks and Synchronization
 
 Validator nodes and fullnodes form a hierarchical structure with validator nodes at the root and fullnodes everywhere else. The Aptos blockchain distinguishes two types of fullnodes: validator fullnodes and public fullnodes. Validator fullnodes connect directly to validator nodes and offer scalability alongside DDoS mitigation. Public fullnodes connect to validator fullnodes (or other public fullnodes) to gain low-latency access to the Aptos network.
 
-![v-fn-network.svg](/img/docs/v-fn-network.svg)
+![v-fn-network.svg](../../static/img/docs/v-fn-network.svg)
```

Comment:
I don't think this change is needed for Aptos.dev:
https://aptos.dev/concepts/basics-node-networks-sync
But should help with GitHub accessibility. Thanks!